app/frontend/src/features/profile/ProfileTagInput.tsx

```diff
@@ -0,0 +1,229 @@
+import React, { useState, useEffect, useRef } from "react";
+import {
+  Checkbox,
+  IconButton,
+  InputBase,
+  ButtonBase,
+  Popper,
+  fade,
+  makeStyles,
+  createStyles,
+} from "@material-ui/core";
+import Autocomplete, {
+  AutocompleteCloseReason,
+} from "@material-ui/lab/Autocomplete";
+import { CloseIcon, ExpandMore } from "../../components/Icons";
+
+const useStyles = makeStyles((theme) =>
+  createStyles({
+    popper: {
+      borderRadius: theme.shape.borderRadius,
+      borderWidth: 1,
+      borderStyle: "solid",
+      borderColor: "rgba(0, 0, 0, 0.23)",
```

Comment:
Is this shade of black likely used across a lot of places? Might be good to extract this into the theme if the answer is yes.

Reply:
This is another value hard-coded in the default outlined input. Definitely should extract at some point. Again, I wasn't sure if it was a good time without a value from design team.